
#include <exception>
#include <iostream>

#include "server_servidor.h"
#define ERROR -1

int main(int argc, char* argv[]) {
    try {
        int ret = ERROR;
        if (argc == 2) {
            Servidor servidor(argv[1]);
            ret = servidor.establecer_conexion();
            servidor.imprimir_cant_acciones();
        } else {
            std::cerr << "Llamada al programa inv치lida. Se espera " << argv[0] << " <servname>\n";
            return ret;
        }
        return ret;
    } catch (const std::exception& err) {
        std::cerr << "Algo sali칩 mal: " << err.what() << "\n";
        return ERROR;
    } catch (...) {
        std::cerr << "Algo sali칩 mal y se intercept칩 un error desconocido.\n";
        return ERROR;
    }
}
